packages/nest/src/utils/versions.ts

```diff
@@ -1,8 +1,8 @@
 export const nxVersion = '*';
 
-export const nestJsVersion = '^7.0.0';
-export const nestJsSchematicsVersion = '^7.0.0';
+export const nestJsVersion = '^8.0.0';
+export const nestJsSchematicsVersion = '^8.0.0';
 
-export const rxjsVersion = '~6.6.3';
+export const rxjsVersion = '^7.2.0';
```

Comment:
How will this interact with Angular (https://github.com/nrwl/nx/blob/master/packages/angular/src/utils/versions.ts#L5) that still requires RxJS 6? I'm asking this because Nx offers an Angular + NestJS workspace by default.

Angular support for RxJS 7 is pending and is expected to arrive in the next major version, ref: https://github.com/angular/angular/issues/41897

Reply:
Yes, indeed this is a problem, with Nest + Next/React should not be any problem thought

Reply:
Yep, confirmed that I upgraded my workspace to Nestjs v8 with RxJS v7 (using React App). Perhaps it could be an opt-in option to set Nestjs v8 (with RxJS v7).